engine/src/dispatch.cpp

```diff
@@ -1130,14 +1138,10 @@ IO_stat MCDispatch::dosavescriptonlystack(MCStack *sptr, const MCStringRef p_fna
         sptr -> unsecurescript(sptr);
         
         // Write out the standard script stack header, and then the script itself
-		MCStringFormat(&t_script_body, "script \"%@\"\n%@", sptr -> getname(), sptr->_getscript());
+        MCStringFormat(&t_script_body, "script \"%@\"\n%@", sptr -> getname(), sptr->_getscript());
 
-		// Convert line endings - but only if the native line ending isn't CR!
-#ifndef __CR__
-		MCStringConvertLineEndingsToLiveCode(*t_script_body, &t_converted);
-#else
-		t_converted = *t_script_body;
-#endif
+        MCStringNormalizeLineEndings(*t_script_body, sptr -> getlineencodingstyle(),
```

Comment:
I suggest making the middle argument on its own line here. There are cases where it makes sense to have multiples ones on the same line (e.g. x, y) when they are related; otherwise separate lines helps readability for long argument lists.